servicecontrol/plugins.md

```diff
@@ -89,6 +89,10 @@ Note that even if an endpoint is able to send heartbeat messages and it is marke
 If a heartbeat message is not received by ServiceControl from an endpoint, that endpoint is marked as "inactive". 
 An inactive endpoint indicates that there is a failure in the communication path between ServiceControl and the monitored endpoint. For example, such failures may be caused by a failure of the endpoint itself, a communication failure in the transport, or when ServiceControl is unable to receive and process the heartbeat messages sent by the endpoint.
 
+
+#### Should I deploy the Heartbeat plugin to Production?
+Please ensure that you **do deploy** this plugin to your endpoint in production. The information sent by this plugin is vital for monitoring. ServicePulse will not be able to indicate if your endpoint is active or not without this information.
```

Comment:
Agreed, perhaps reword to point out the benefits of using it and let the users realized that they "should deploy" it them selves?